meshtastic/__init__.py

```diff
@@ -31,6 +31,7 @@
 - meshtastic.receive.user(packet)
 - meshtastic.receive.data.portnum(packet) (where portnum is an integer or well known PortNum enum)
 - meshtastic.node.updated(node = NodeInfo) - published when a node in the DB changes (appears, location changed, username changed, etc...)
+- meshtastic.log.line(line) - a raw unparsed log line from the radio
```

Comment:
I was mentioning before, I think I might change this, if that's okay (would do after merge, and I'd update the spot you've used it as well) -- I'd be changing it to pass the whole `LogRecord` as a dict, so that folks can pull out the message, level, and time/source (if those are provided) separately and do with them as they will. This will also mean I can have `ble_interface` output this same message and move all the log printing/colorizing stuff to one place, hopefully.

Reply:
yes - I agree, LogRecord is what should be passed (using the automatic KnownProtocol translation)